src/RageSound.cpp

```diff
@@ -288,6 +288,10 @@ int RageSound::GetDataToPlay( float *pBuffer, int iFrames, std::int64_t &iStream
 		/* Read data from our source. */
 		int iGotFrames = m_pSource->RetriedRead( pBuffer + (iFramesStored * m_pSource->GetNumChannels()), iFrames, &iSourceFrame, &fRate );
 
+#ifdef ERROR
```

Comment:
Sure.

Reply:
`windows.h` isn't used in RageSound, it's pulled in via GameLoop. The reason the undef is located here is to prevent a compiler error about ERROR already being defined. So if I undef ERROR in GameLoop it still needs to be undef'd here.

However.... if we look up the definition of `RageSoundReader::ERROR`, it equals `-1`, so any reason not to just change line 291 to look for `-1`?